CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
-ChangeLog
----------
+# Change Log
 
-All notable changes to this project will be documented in this file.
+All notable changes to this project will be documented in this file
+automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-### v5.0.22
+## v5.0.22
```

Comment:
It's down to `versionist` and I don't think it cares as long as it finds the latest and the tag.